sonic-ycabled/tests/test_y_cable_helper.py

```diff
@@ -192,9 +192,19 @@ def test_y_cable_wrapper_get_presence_with_platform_chassis_raise_exception(self
                                                                                                'version_peer_inactive': '1.7MS',
                                                                                                'version_peer_next': '1.7MS'}))
     def test_post_port_mux_info_to_db(self):
+        asic_index = 0
         logical_port_name = "Ethernet0"
-        mux_tbl = Table("STATE_DB", "Y_CABLE_INFO_TABLE")
-        rc = post_port_mux_info_to_db(logical_port_name, mux_tbl,'active-standby')
```

Comment:
can we add a test to crash at known location and see if we print backtrace ?

Reply:
UT is added, fixed